doc/mgr/dashboard.rst

```diff
@@ -236,6 +236,12 @@ exist and you may find yourself in the situation that you have to use them::
   $ ceph dashboard set-rgw-api-admin-resource <admin_resource>
   $ ceph dashboard set-rgw-api-user-id <user_id>
 
+If you are using a self-signed certificate in your Object Gateway setup, then
+you should disable certificate verification in the dashboard to avoid refused
+connections::
+
+  $ ceph dashboard set-rgw-api-no-ssl-verify True
```

Comment:
I'd suggest to rename this to `rgw-api-cert-verification`. You're verifying the certificate, not the SSL protocol per se :)

Also, I think we should avoid double negation to avoid confusion: instead of setting "no-ssl-verify" to "True", I suggest to use set the verification to "False" (disabled).

Reply:
I've adapted to name from various tools that support a know called ``--no-ssl-verify``:

https://git-scm.com/docs/git-config#git-config-httpsslVerify
https://github.com/VundleVim/Vundle.vim/wiki#faq5
https://puppet.com/docs/bolt/0.x/new_features.html#support-for-no-ssl-verify-flag-0-18-0
https://phab.enlightenment.org/w/efl.net/
https://www.icinga.com/docs/director/latest/vsphere/doc/04-CLI-Commands/

Reply:
I'd also prefer positive names for boolean variables and I think that it's a difference if a command line flag is called `--no-ssl-verify` which disables a features without having to provide a value than using a negated variable name and having to provide a value.

http://truelogic.org/wordpress/2009/02/27/how-not-to-name-a-boolean-variable/
https://stackoverflow.com/questions/1227998/naming-conventions-what-to-name-a-boolean-variable
http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification